Traceback (most recent call last):
  File "/tmp/tmpvafra3c0", line 5, in <module>
    with open('270_text.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: '270_text.txt'